tanner/tests/test_rfi_emulation.py

```diff
@@ -22,10 +22,8 @@ def test_http_download_fail(self):
         self.assertIsNone(filename)
 
     def test_ftp_download(self):
-        self.handler.download_file_ftp = mock.MagicMock()
         path = 'ftp://mirror.yandex.ru/archlinux/lastupdate'
-        data = self.loop.run_until_complete(self.handler.download_file(path))
-        self.handler.download_file_ftp.assert_called_with(yarl.URL(path))
+        result = self.loop.run_until_complete(self.handler.download_file_ftp(yarl.URL(path)))
```

Comment:
don't change the function call. For real request we call `download_file` function and we should check that it works correctly with ftp